### PR TITLE
DEVDOCS-5810 [update]: Tax Provider API, update response body for adjust operation

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-5810]

## What changed?
* Update response body schema for adjust operation
* Remove incorrect response example for adjust operation

[DEVDOCS-5810]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-5810?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: reference/tax_provider.yml [hunk 1]
```diff
@@ -614,113 +614,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/request-adjust'
-              examples:
```
Comment (bc-andreadao) [inline reference/tax_provider.yml:618]: I removed the incorrect example, but if there's a good example we could use, we could add it in

Comment (theromulans) [reply reference/tax_provider.yml:618]: It’d be the same example as any of the others, although we could get fancy with it I suppose. I think it’s probably fine to simplify.

**Review by theromulans — DISMISSED** [review]

**Comment by bc-andreadao** [comment]
Thanks @theromulans

**Review by bc-traciporter — APPROVED** [review]
LGTM!

**Pull request merged**